concepts/Common/Security Considerations/20 HTML Encoding/30 Potentially Dangerous API Members/0 Potentially Dangerous API Members.md

```diff
@@ -0,0 +1 @@
+DevExtreme components encode HTML in most cases. However, several components include API members that allow you to disable or bypass the encoding. The following sections describe these potentially vulnerable API members.
```

Comment:
"DevExtreme components encode HTML in most cases." - This is a vague statement since we don't specify the cases.

```suggestion
Several components include API members that allow you insert unencoded HTML. The following sections describe these potentially vulnerable API members.
```